source/docs/software/advanced-gradlerio/deploy-git-data.rst

```diff
@@ -1,113 +1,36 @@
 
-.. note:: Git must be in the path for this to work. This should be enabled by default when `installing Git <https://git-scm.com/downloads>`__.
+- Installing gversion
 
-Deploying Branch Name
----------------------
-
-This example uses `git rev-parse <https://git-scm.com/docs/git-rev-parse>`__ to extract data the name of the current branch. The Git command used for this is:
-
-.. code-block:: console
-
-   $ git rev-parse --abbrev-ref HEAD
-
-The ``--abbrev-ref`` flag tells Git to use a short version of the name for the current commit that ``rev-parse`` is acting on. When HEAD is the most recent commit on a branch, this will return the name of that branch.
-
-Next, create a new task in the ``build.gradle`` file that will run the above Git command and write it to a file in the ``src/main/deploy`` directory. For example, the following is an example task named ``writeBranchName`` that will write the branch name to a file named ``branch.txt``.
+To install gversion add the following line to the plugins block of `build.gradle`.
 
 .. code-block:: groovy
 
-   tasks.register("writeBranchName") {
-      // Define an output stream to write to instead of terminal
-      def stdout = new ByteArrayOutputStream()
-
-      // Execute the git command
-      exec {
-         commandLine "git", "rev-parse", "--abbrev-ref", "HEAD"
-         // Write to the output stream instead of terminal
-         standardOutput = stdout
-      }
-
-      // Parse the output into a string
-      def branch = stdout.toString().trim()
-
-      // Create a new file
-      new File(
-         // Join project directory and deploy directory
-         projectDir.toString() + "/src/main/deploy",
-         // File name to write to
-         "branch.txt"
-      ).text = branch // Set the contents of the file to the variable branch
-   }
+  plugins {
+      // ...
+      id "com.peterabeles.gversion" version "1.10"
+  }
 
-This registers a `Gradle task <https://docs.gradle.org/current/userguide/tutorial_using_tasks.html>`__ that uses the above Git command, saves the output to a variable, and then writes it to a file. Since it was written to the ``src/main/deploy`` directory, it will be included in the jar file deployed to the robot and accessible in code.
-
-The next step is to make the deploy task depend on the task you created, so that it will automatically run before the code is deployed. This example uses the task name ``writeBranchName`` from the previous example, but it should be replaced with the name of the task in your ``build.gradle``.
+Then add the following block to `build.gradle` to add `createVersionFile` as a dependancy of `compileJava`. The `timeZone` field can be modified based on your team's time zone based on `this list of timezone IDs <https://docs.oracle.com/middleware/12211/wcs/tag-ref/MISC/TimeZones.html>`__.
```

Comment:
Explain what this does- similar to the previous contents that said "This registers a Gradle task that uses the above Git command...."